packages/puppeteer-core/src/bidi/Realm.ts

```diff
@@ -179,32 +141,71 @@ export class BidiRealm extends EventEmitter<Record<EventType, any>> {
       functionDeclaration = SOURCE_URL_REGEX.test(functionDeclaration)
         ? functionDeclaration
         : `${functionDeclaration}\n${sourceUrlComment}\n`;
-      responsePromise = this.connection.send('script.callFunction', {
-        functionDeclaration,
+      responsePromise = this.realm.callFunction(functionDeclaration, true, {
```

Comment:
Is there a reason why awaitPromise is not part of options? it makes it less readable. At least let's add a comment `/*awaitPromise=*/` or move it to the options

Reply:
Since required options always have to be added, my goal here is to have required options are inlined as parameters while optional ones are put into an options object. Anyway, done!